pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>com.clevertap.apns</groupId>
     <artifactId>apns-http2</artifactId>
-    <version>1.0.3</version>
+    <version>2.0</version>
```

Comment:
Where does to `2.0` come from? There is no `2.0` release.

Reply:
it is our version

Reply:
As discussed, create a PR to the original repository to at least try to keep the versions in sync :)